libs/portal-integration-angular/src/lib/core/components/portal-header/header.component.html

```diff
@@ -197,7 +197,7 @@ <h5 class="flex flex-row align-items-center justify-content-center text-color-se
             <ng-container *ngFor="let item of moreMenuItems">
               <li class="layout-topbar-action-item" *ocxIfPermission="item.permission">
                 <!-- *ngIf="!item?.disabled" -->
-                <a
+                <!-- <a
```

Comment:
Why?

Reply:
forgot to revert, because I get the error: Type 'MouseEvent' has no properties in common with type 'MenuItemCommandEvent' that I couldn't resolve